mpxe/integration_tests/init_cond.py

```diff
@@ -0,0 +1,55 @@
+from mpxe.protogen import mcp23008_pb2, pca9539r_pb2
```

Comment:
The functions in this file should be per integration test, so I don't think they should be in a separate file.